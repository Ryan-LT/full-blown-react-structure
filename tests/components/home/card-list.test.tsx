import React from 'react';
import { CardList } from '@components';
import { cleanup, render } from '../../renderTest';
import '@testing-library/jest-dom';

afterEach(cleanup);

it('should take a snapshot', () => {
  const { asFragment } = render(<CardList />);

  expect(asFragment()).toMatchSnapshot();
});

it('should render without issue', () => {
  const component = render(<CardList />);
  expect(component).toBeTruthy();
});
