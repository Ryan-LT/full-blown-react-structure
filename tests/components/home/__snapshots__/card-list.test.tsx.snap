// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="data-list"
  />
</DocumentFragment>
`;
